packages/utils/src/lib/load-file.ts

```diff
@@ -1,5 +1,17 @@
 import { type Options, bundleRequire } from 'bundle-require';
 
+export class NoExportError extends Error {
+  constructor(filepath: string) {
+    super(`No export found in ${filepath}`);
+  }
+}
+
+export class NoFileError extends Error {
+  constructor(filepath: string) {
+    super(`The entry point "${filepath}" cannot be marked as external`);
+  }
+}
```

Comment:
Is this used anywhere?

Reply:
It was used in a test before @Tlacenka pointed out the test will pass without the error 😅

Reply:
removed